data/ulx/config.txt
;Any of the settings in here can be added to the per-map or per-gamemode configs.
;To add per-map and per-gamemode configs, create data/ulx/maps/<mapname>/config.txt
;and data/ulx/gamemodes/<gamemodename>/config.txt files. This can also be done for
;All other configuration files (adverts.txt, downloads.txt, gimps.txt, votemaps.txt)
;All configurations add to each other except gimps and votemaps, which takes the most
;specific config only.
;Any line starting with a ';' is a comment!

ulx showMotd 0 ; MOTD mode
; MOTD modes:
; 0 - OFF No MOTD shown
; 1 - FILE Show the players the contents of the file from the 'motdfile' cvar
; 2 - GENERATOR Uses the MOTD generator to create a MOTD for the player (use XGUI for this)
; 3 - URL Show the player the URL specified by the 'motdurl' cvar
; In a URL, you can use %curmap% and %steamid% in the URL to have it automagically parsed for you (eg, server.com/?map=%curmap%&id=%steamid%).
ulx motdfile ulx_motd.txt ; The MOTD to show, if using a file. Put this file in the root of the garry's mod directory.
ulx motdurl ulyssesmod.net ; The MOTD to show, if using a URL.


ulx chattime 0 ; Players can only chat every x seconds (anti-spam). 0 to disable
ulx meChatEnabled 1 ; Allow players to use '/me' in chat. 0 = Disabled, 1 = Sandbox only (Default), 2 = Enabled


; This is what the players will see when they join, set it to "" to disable.
; You can use %host% and %curmap% in your text and have it automagically parsed for you
ulx welcomemessage "Welcome to %host%! We're playing %curmap%." 


ulx logFile 1 ; Log to file (Can still echo if off). This is a global setting, nothing will be logged to file with this off.
ulx logEvents 1 ; Log events (player connect, disconnect, death)
ulx logChat 1 ; Log player chat
ulx logSpawns 1 ; Log when players spawn objects (props, effects, etc)
ulx logSpawnsEcho 1 ; Echo spawns to players in server. -1 = Off, 0 = Dedicated console only, 1 = Admins only, 2 = All players. (Echoes to console)
ulx logJoinLeaveEcho 1 ; Echo players leaves and joins to admins in the server (useful for banning minges)
ulx logDir ulx_logs ; The log dir under garrysmod/data
ulx logEcho 2 ; Echo mode
; Echo modes:
; 0 - OFF No output to any players when an admin command is used
; 1 - ANONYMOUS Output to players without access to see who used the command (admins by default) similar to "(Someone) slapped Bob with 0 damage"
; 2 - FULL Output to players similar to "Foo slapped Bob with 0 damage"

ulx logEchoColors 1 ; Whether or not echoed commands in chat are colored
ulx logEchoColorDefault "151 211 255" ; The default text color (RGB)
ulx logEchoColorConsole "0 0 0" ; The color that Console gets when using actions
ulx logEchoColorSelf "75 0 130" ; The color for yourself in echoes
ulx logEchoColorEveryone "0 128 128" ; The color to use when everyone is targeted in echoes
ulx logEchoColorPlayerAsGroup 1 ; Whether or not to use group colors for players. If false, it uses the color below.
ulx logEchoColorPlayer "255 255 0" ; The color to use for players when ulx logEchoColorPlayerAsGroup 1 
ulx logEchoColorMisc "0 255 0" ; The color for anything else in echoes

ulx rslotsMode 0 
ulx rslots 2 
ulx rslotsVisible 1 ; When this is 0, sv_visiblemaxplayers will be set to maxplayers - slots_currently_reserved
;Modes:
;0 - Off
;1 - Keep # of slots reserved for admins, admins fill slots.
;2 - Keep # of slots reserved for admins, admins don't fill slots, they'll be freed when a player leaves.
;3 - Always keep 1 slot open for admins, kick the user with the shortest connection time if an admin joins.

;Difference between 1 and 2:
;I realize it's a bit confusing, so here's an example.
;On mode 1--
;	You have maxplayers set to 10, rslots 2 
;	If a non-admin tries to join, they'll be kicked to keep the reserved slots open. Two admins join
;	and fill the two reserved slots. When non-admins leave, the two admins will still be filling the
;	two reserved slots, so another regular player can join and fill the server up again without being
;	kicked by the slots system

;On mode 2--
;	Same setup as mode 1, you have the two admins in the server and the server is full. Now, when a
;	non-admin leaves the server, reserved slots will pick up the slot again as reserved. If a regular
;	player tries to join and fill the server again, even though there are two admins connected, it will
;	kick the regular player to keep the slot open

;So, the basic difference between these two is mode 1 will subtract currently connected admins from the slot
;pool, while mode 2 while always be attempting to reclaim slots if it doesn't currently have enough when
;players leave no matter how many admins are connected.

;rslotsVisible:
;	If you set this variable to 0, ULX will automatically change sv_visiblemaxplayers for you so that if
;	there are no regular player slots available in your server, it will appear that the server is full.
;	The major downside to this is that admins can't connect to the server using the "find server" dialog
;	when it appears full. Instead, they have to go to console and use the command "connect <ip>".
;	NOTE THIS DOES NOT CHANGE YOUR MAXPLAYERS VARIABLE, ONLY HOW MANY MAXPLAYERS IT _LOOKS_ LIKE YOUR
;	SERVER HAS. YOU CAN NEVER, EVER HAVE MORE PLAYERS IN YOUR SERVER THAN THE MAXPLAYERS VARIABLE.



ulx votemapEnabled 1 ; Enable/Disable the entire votemap command
ulx votemapMintime 10 ; Time after map change before votes count.
ulx votemapWaittime 5 ; Time before a user must wait before they can change their vote.
ulx votemapSuccessratio 0.4 ; Ratio of (votes for map)/(total players) needed to change map. (Rounds up)
ulx votemapMinvotes 3 ; Number of minimum votes needed to change map (Prevents llamas). This supercedes the above convar on small servers.
ulx votemapVetotime 30 ; Time in seconds an admin has after a successful votemap to veto the vote. Set to 0 to disable.
ulx votemapMapmode 1 ; 1 = Use all maps but what's specified in votemaps.txt, 2 = Use only the maps specified in votemaps.txt.

ulx voteEcho 0 ; 1 = Echo what every player votes (this does not apply to votemap). 0 = Don't echo

ulx votemap2Successratio 0.5 ; Ratio of (votes for map)/(total players) needed to change map. (Rounds up)
ulx votemap2Minvotes 3 ; Number of minimum votes needed to change map (Pevents llamas). This supercedes the above convar on small servers.

ulx votekickSuccessratio 0.6 ; Ratio of (votes for kick)/(total players) needed to kick player. (Rounds up)
ulx votekickMinvotes 2 ; Number of minimum votes needed to kick player (Pevents llamas). This supercedes the above convar on small servers.

ulx votebanSuccessratio 0.7 ; Ratio of (votes for ban)/(total players) needed to ban player. (Rounds up)
ulx votebanMinvotes 3 ; Number of minimum votes needed to ban player (Pevents llamas). This supercedes the above convar on small servers.
ulx kickAfterNameChanges 0 ; <number> - Players can only change their name x times every ulx_kickAfterNameChangesCooldown 60 
ulx kickAfterNameChangesWarning 1 ; <1/0> - Display a warning to users to let them know how many more times they can change their name.


data/mapvote/config.txt
{"RTVPlayerCount":3.0,"AutoGamemode":false,"MapLimit":24.0,"TimeLimit":28.0,"AllowCurrentMap":false,"MapPrefixes":["ttt_"],"MapsBeforeRevote":10.0,"EnableCooldown":true}

addons/1darkrpmodification/lua/darkrp_config/settings.lua
-- voice3D - Включить/Выключить 3D голос
GM.Config.voice3D    = true
-- AdminsCopWeapons - Включить/Выключить спавн админов с оружием полиции
GM.Config.AdminsCopWeapons = false
-- adminBypassJobCustomCheck - включить/отключить, может ли администратор принудительно установить задание с всякий раз, когда customCheck возвращает false
GM.Config.adminBypassJobRestrictions    = true
-- Разрешить игрокам использовать команду /job
GM.Config.allowjobswitch = false
-- allowrpnames - Разрешить игрокам устанавливать себе РП-имя через команду /rpname
GM.Config.allowrpnames = true
-- allowsprays - Включить/Выключить возможность использовать спреи на сервере.
GM.Config.allowsprays = true
-- allowvehicleowning - Включить/Выключить возможность владения машинами.
GM.Config.allowvehicleowning = true
-- allowvnocollide - Включить/Выключить столкновения с машиной (для защиты).
GM.Config.allowvnocollide = false
-- alltalk - Значение true для глобального чата, значение false для локального чата.
GM.Config.alltalk = false
-- autovehiclelock - Включить/Выключить автоматическое закрывание машины, когда игрок выйдет из неё.
GM.Config.autovehiclelock = false
-- babygod - игроки спавнятся с бессмертием (защита от убийств на спавне)
GM.Config.babygod = false
-- canforcedooropen - могут ли игроки заставить неуправляемую дверь открыться с помощью отмычки или тарана или ж / д
GM.Config.canforcedooropen    = true
-- chatsounds - проигрывать звук, когда кто-то говорил в чат
GM.Config.chatsounds    = true
-- chiefjailpos - Разрешает шефу полиции устанавливать тюремные позиции.
GM.Config.chiefjailpos = false
-- cit_propertytax - включить / отключить налог на имущество, который является эксклюзивным только для граждан.
GM.Config.cit_propertytax = false
-- copscanunfreeze - включение / отключение способности полицейских размораживать чужой реквизит
GM.Config.copscanunfreeze = true
-- copscanunweld - включение / отключение способности полицейских разматывать чужой реквизит
GM.Config.copscanunweld = false
-- cpcanarrestcp - Включить/Выключить возможность арестовать полицейского.
GM.Config.cpcanarrestcp = false
-- currencyLeft - Позиция символа валюты. true - слева, false - справа.
GM.Config.currencyLeft    = false
-- customjobs - Включить/Выключить команду /job (персональные названия работ).
GM.Config.customjobs = false
-- customspawns - Включить/Выключить использование нестандартных точек спавна.
GM.Config.customspawns = false
-- deathblack - Затемнение экрана после смерти.
GM.Config.deathblack = true
-- showdeaths - отображение информации об убийстве в правом верхнем углу экрана каждого человека.
GM.Config.showdeaths = false
-- deadtalk - Включить/Выключить возможность использования комманд и чата, когда игрок мертв
GM.Config.deadtalk = false
-- deadvoice - Включить/Выключить возможность говорить в микрофон, когда игрок мертв
GM.Config.deadvoice    = false
-- deathpov - Включить/Выключить возможность видеть от первого лица, когда игрока убили
GM.Config.deathpov = true
-- decalcleaner - Включить/Выключить удаление декалей игроков (спреев).
GM.Config.decalcleaner = true
-- disallowClientsideScripts - лиентские скрипты могут быть очень полезны для настройки HUD или для помощи в построении. Этот параметр запрещает эти сценарии.
GM.Config.disallowClientsideScripts    = false
-- doorwarrants - Включить / отключить требование ордера для входа в собственность.
GM.Config.doorwarrants = true
-- dropmoneyondeath - Включить/Выключить выпадение денег при смерти игрока.
GM.Config.dropmoneyondeath = true
-- droppocketarrest - Включить/Выключить выпадение вещей из инвенторя при аресте игрока.
GM.Config.droppocketarrest = false
-- droppocketdeath - Включить/Выключить выпадение вещей из инвенторя при смерти игрока.
GM.Config.droppocketdeath = true
-- dropweapondeath - Включить/Выключить выпадение текущего оружия при смерти игрока.
GM.Config.dropweapondeath = true
-- Включить/Выключить возможность выкинуть стандартное оружие (с которым появился)
GM.Config.dropspawnedweapons    = false
-- dynamicvoice - выключение того, могут ли только люди в той же комнате, что и вы, слышать ваш микрофон.
GM.Config.dynamicvoice = true
-- earthquakes - Включить/Выключить землетрясения.
GM.Config.earthquakes = false
-- enablebuypistol - Включить/Выключить команду /buy
GM.Config.enablebuypistol = true
-- enforceplayermodel - следует ли заставлять игроков использовать свои ролевые модели персонажей или нет.
GM.Config.enforceplayermodel = true
-- globalshow - Включить/Выключить показ информации об игроке над головой.
GM.Config.globalshow = false
-- ironshoot - включение / выключение того, нужны ли людям железные прицелы для стрельбы.
GM.Config.ironshoot = true
-- showjob - Включить/Выключить показ информации о работе игрока над головой.
GM.Config.showjob = true
-- letters - Включить/Выключить возможность писать записки.
GM.Config.letters = true
-- license - Включить/Выключить необходимость иметь лицензию, чтобы взять оружие
GM.Config.license = false
-- lockpickfading - включение / отключение взлом замков исчезающих дверей
GM.Config.lockpickfading    = true
-- logging - Включить/Выключить запись логов о происходящем на сервере.
GM.Config.logging = true
-- lottery - Включить/Выключить возможность создания лотерее мэром
GM.Config.lottery = true
-- showname - Включить/Выключить показ имени игрока над головой.
GM.Config.showname = false
-- showname - Включить/Выключить показ здоровья игрока над головой.
GM.Config.showhealth    = false
-- needwantedforarrest - Включить/Выключить арест игроков только если они в розыске.
GM.Config.needwantedforarrest = false
-- noguns - включение этой функции запрещает оружие и торговцев оружием.
GM.Config.noguns = false
-- norespawn - включение / выключение того, что людям не нужно возрождаться, когда они меняют работу.
GM.Config.norespawn = true
-- npcarrest - Включить/Выключить возможность арестовывать NPC
GM.Config.npcarrest = false
-- ooc - Включить/Выключить использование OOC.
GM.Config.ooc = true
-- propertytax - включение / отключение налога на имущество.
GM.Config.propertytax = false
-- proppaying - независимо от того, должны ли игроки платить за нерест реквизита.
GM.Config.proppaying = false
-- propspawning - включение / выключение нереста реквизитов для не-администраторов.
GM.Config.propspawning = true
-- removeclassitems - включение/отключение поставок/микроволновая печь/и т. д. удаление, когда кто-то меняет команду.
GM.Config.removeclassitems = true
-- respawninjail - включение / выключение того, могут ли люди возрождаться в тюрьме, когда они умирают
GM.Config.respawninjail = true
-- restrictallteams - включение/отключение игроки могут только быть гражданином до тех пор, пока админ разрешает их.
GM.Config.restrictallteams = false
-- restrictbuypistol - включение этой функции делает / buy доступным только для торговцев оружием.
GM.Config.restrictbuypistol = false
-- restrictdrop - включение / выключение ограничения оружия, которое игроки могут уронить. Установка этого значения в true запрещает сбрасывать оружие из поставок
GM.Config.restrictdrop = false
-- revokeLicenseOnJobChange - аннулируются ли лицензии, когда игрок меняет задания
GM.Config.revokeLicenseOnJobChange    = true
-- shouldResetLaws - включить / отключить сброс законов обратно к закону по умолчанию, установленному при изменении мэра
GM.Config.shouldResetLaws = false
-- strictsuicide - должны ли игроки появляться там, где они покончили с собой
GM.Config.strictsuicide = false
-- telefromjail - Включить/Выключить телепортирование из тюрьмы.
GM.Config.telefromjail = true
-- teletojail - Включить/Выключить телепортирование в тюрьму.
GM.Config.teletojail = true
-- unlockdoorsonstart - Включить/Выключить открывание открывание всех дверей на карте при старте сервера.
GM.Config.unlockdoorsonstart = false
-- voiceradius - Включить/Выключить локальный голосовой чат.
GM.Config.voiceradius = true
-- tax - платят ли игроки налоги на свои кошельки.
GM.Config.wallettax = false
-- wantedsuicide - включить / отключить самоубийство, пока вы находитесь в розыске полицией.
GM.Config.wantedsuicide = false
-- realisticfalldamage - включение / выключение динамического урона от падения. Установка mp_falldamage 1, будут более-ездить.
GM.Config.realisticfalldamage    = true
-- printeroverheat - может ли принтер денег по умолчанию перегреваться сам по себе?
GM.Config.printeroverheat    = true
-- weaponCheckerHideDefault - скрыть оружие по умолчанию при проверке оружия
GM.Config.weaponCheckerHideDefault    = true
-- weaponCheckerHideNoLicense - скрыть оружие, которое не требует лицензии
GM.Config.weaponCheckerHideNoLicense    = false
--[[
Value settings
]]
-- adminnpcs - Whether or not NPCs should be admin only. 0 = everyone, 1 = admin or higher, 2 = superadmin or higher, 3 = rcon only
GM.Config.adminnpcs                     = 2
-- adminsents - Whether or not SENTs should be admin only. 0 = everyone, 1 = admin or higher, 2 = superadmin or higher, 3 = rcon only
GM.Config.adminsents                    = 2
-- adminvehicles - Whether or not vehicles should be admin only. 0 = everyone, 1 = admin or higher, 2 = superadmin or higher, 3 = rcon only
GM.Config.adminvehicles                 = 2
-- adminweapons - Who can spawn weapons: 0: admins only, 1: supadmins only, 2: no one
GM.Config.adminweapons                  = 1
-- arrestspeed - устанавливает максимальную скорость ареста..
GM.Config.arrestspeed                   = 120
-- babygodtime - ак долго длится babygod.
GM.Config.babygodtime                   = 5
-- chatsoundsdelay - как долго нужно ждать, прежде чем позволить игроку снова издавать звук из своего Чата.
-- Оставьте это включенным хотя бы на несколько секунд, чтобы люди не спамили звуки. Установите значение 0, чтобы отключить его.
GM.Config.chatsoundsdelay               = 5
-- deathfee - сумма денег, которую кто-то роняет, когда умирает.
GM.Config.deathfee                      = 15
-- maxadvertbillboards - The maximum number of /advert billboards a player can place.
GM.Config.maxadvertbillboards           = 0
-- decaltimer - устанавливает время для очистки надписей на стороне клиента 
GM.Config.decaltimer                    = 300000
-- maxCheques - The maximum number of cheques someone can write
GM.Config.maxCheques                    = 0
-- demotetime - количество секунд до того, как игрок сможет присоединиться к команде после понижения в должности из этой команды.
GM.Config.demotetime                    = 300
-- doorcost - устанавливает стоимость двери.
GM.Config.doorcost                      = 5
-- entremovedelay - колько времени нужно ждать перед удалением купленного объекта после отключения.
GM.Config.entremovedelay                = 0
-- gunlabweapon - оружие, которое порождает gunlab.
GM.Config.gunlabweapon                  = "weapon_p2282"
-- jailtimer - устанавливает jailtimer (in seconds).
GM.Config.jailtimer                     = 120
-- lockdowndelay - количество времени, которое мэр должен ждать, прежде чем начать следующую блокировку.
GM.Config.lockdowndelay                 = 120
-- maxdoors - устанавливает максимальное количество дверей, которыми можно владеть.
GM.Config.maxdoors                      = 6
-- maxdrugs - Sets max drugs.
GM.Config.maxdrugs                      = 0
-- maxfoods - устанавливает максимальное количество пищевых коробок для каждого владельца микроволновой печи.
GM.Config.maxfoods                      = 2
-- maxlawboards - максимальное количество юридических досок, которые может разместить мэр.
GM.Config.maxlawboards                  = 2
-- maxletters - устанавливает максимальное количество букв..
GM.Config.maxletters                    = 10
-- maxlotterycost - максимальный платеж, который мэр может установить, чтобы присоединиться к лотерее.
GM.Config.maxlotterycost                = 10000
-- maxvehicles - устанавливает, сколько транспортных средств можно купить.
GM.Config.maxvehicles                   = 5
-- microwavefoodcost - станавливает цену продажи продуктов питания в микроволновой печи.
GM.Config.microwavefoodcost             = 30
-- minlotterycost - минимальный платеж, который мэр может установить, чтобы присоединиться к лотерее.
GM.Config.minlotterycost                = 500
-- Money packets will get removed if they don't get picked up after a while. Set to 0 to disable.
GM.Config.moneyRemoveTime               = 1200
-- mprintamount - стоимость денег, напечатанных на денежном принтере.
GM.Config.mprintamount                  = 250
-- normalsalary - устанавливает начальную зарплату для вновь присоединившихся игроков.
GM.Config.normalsalary                  = 0
-- npckillpay - устанавливает деньги, полученные за каждое убийство NPC.
GM.Config.npckillpay                    = 0
-- paydelay - устанавливает, сколько времени требуется, чтобы люди получили зарплату.
GM.Config.paydelay                      = 600
-- pocketitems - устанавливает количество предметов, которые карман может нести.
GM.Config.pocketitems                   = 10
-- pricecap - максимальная цена товара (using /price).
GM.Config.pricecap                      = 500
-- pricemin - минимальная цена товара (using /price).
GM.Config.pricemin                      = 50
-- propcost - сколько должен стоить нерест prop (prop paying должен быть включен, чтобы это имело эффект).
GM.Config.propcost                      = 10
-- quakechance - вероятность возникновения землетрясения.
GM.Config.quakechance                   = 4000
-- respawntime - минимальное количество секунд, которые игрок должен ждать перед респауном.
GM.Config.respawntime                   = 1
-- changejobtime - минимальное количество секунд, которое игрок должен ждать перед сменой задания.
GM.Config.changejobtime                 = 180
-- runspeed - устанавливает максимальную скорость бега.
GM.Config.runspeed                      = 240
-- runspeed - устанавливает максимальную скорость бега для команд CP.
GM.Config.runspeedcp                    = 255
-- searchtime - время поиска-количество секунд, в течение которых действует ордер на обыск.
GM.Config.searchtime                    = 180
-- ShipmentSpawnTime - время защиты от спама между нерестовыми поставками.
GM.Config.ShipmentSpamTime              = 3
-- shipmenttime - время отгрузки-количество секунд, которое требуется для того, чтобы груз появился на свет.
GM.Config.shipmentspawntime             = 10
-- startinghealth - the health when you spawn.
GM.Config.startinghealth                = 100
-- startingmoney - ваш кошелек, когда вы присоединяетесь в первый раз.
GM.Config.startingmoney                 = 100
-- vehiclecost - устанавливает стоимость транспортного средства (для его владения).
GM.Config.vehiclecost                   = 40
-- wallettaxmax - максимальный процент налога, подлежащий уплате.
GM.Config.wallettaxmax                  = 5
-- wallettaxmin - минимальный процент налога, подлежащий уплате.
GM.Config.wallettaxmin                  = 1
-- wallettaxtime - время в секундах между налоговыми игроками. Требуется перезагрузка сервера.
GM.Config.wallettaxtime                 = 600
-- wantedtime - количество секунд, в течение которых игрок находится в розыске.
GM.Config.wantedtime                    = 0
-- walkspeed - устанавливает максимальную скорость ходьбы.
GM.Config.walkspeed                     = 160
-- falldamagedamper - демпфер на реалистичном уроне от падения. Значение по умолчанию-15. Уменьшите это для большего урона.
GM.Config.falldamagedamper              = 15
-- falldamageamount - Базовый урон, полученный от падения за статический урон от падения. Значение по умолчанию-10.
GM.Config.falldamageamount              = 10
-- printeroverheatchance - вероятность перегрева принтера. Чем выше это число, тем меньше вероятность (минимум 3, по умолчанию 22).
GM.Config.printeroverheatchance         = 22
-- printerreward - награда за уничтожение денежного принтера.
GM.Config.printerreward                 = 950
--[[---------------------------------------------------------------------------
Прочие настройки
---------------------------------------------------------------------------]]
-- The classname of money packets. Use this to create your own money entity!
-- Note: the money packet must support the "Setamount" method (or the amount DTVar)
GM.Config.MoneyClass = "spawned_money"
-- In case you do wish to keep the default money, but change the model, this option is the way to go:
GM.Config.moneyModel = "models/props/cs_assault/money.mdl"

-- The skin DarkRP uses. Set to "default" to use the GMod default derma theme.
GM.Config.DarkRPSkin = "DarkRP"
GM.Config.currency = " KWD"
GM.Config.chatCommandPrefix = "/"
GM.Config.F1MenuHelpPage = "http://wiki.darkrp.com/index.php/Main_Page"
GM.Config.F1MenuHelpPageTitle = "Wiki page"
-- Put Steam ID's and ranks in this list, and the players will have that rank when they join.
GM.Config.DefaultPlayerGroups = {
	["STEAM_0:0:00000000"] = "superadmin",
	["STEAM_0:0:11111111"] = "admin",
}
-- Custom modules in this addon that are disabled.
GM.Config.DisabledCustomModules = {
	["hudreplacement"] = false,
	["extraf4tab"] = false,
}
-- The list of weapons that players are not allowed to drop. Items set to true are not allowed to be dropped
GM.Config.DisallowDrop = {
	["arrest_stick"] = true,
	["door_ram"] = true,
	["gmod_camera"] = true,
	["gmod_tool"] = true,
	["keys"] = true,
	["lockpick"] = true,
	["med_kit"] = true,
	["pocket"] = true,
	["stunstick"] = true,
	["unarrest_stick"] = true,
	["weapon_keypadchecker"] = true,
	["weapon_physcannon"] = true,
	["weaponchecker"] = true,
	["weapon_smallcombineshield"] = true,
	["zombieswep"] = true,
	["weapon_new_stunstick"] = true,
	["swep_vortigaunt_beam"] = true,
	["grub_combine_sniper"] = true,
	["med_kit"] = true,
	["weapon_physgun"] = true,
 }
-- Список оружий, с которыми появляются игроки
GM.Config.DefaultWeapons = {
	"",
	"",
	"",
	"",
	"",
	""
}
-- Перезапись категорий
-- NOTE: categories are to be set in the "category" field of the custom jobs/shipments/entities/ammo/pistols/vehicles
-- Use this only to override the categories of _default_ things.
-- This will NOT work for your own custom stuff.
-- Make sure the category is created in the darkrp_customthings/categories.lua, otherwise it won't work!
GM.Config.CategoryOverride = {
jobs = {
	["Citizen"] = "Citizens",
	["Hobo"] = "Citizens",
	["Gun Dealer"] = "Citizens",
	["Medic"] = "Citizens",
	["Civil Protection"] = "Civil Protection",
	["Gangster"] = "Gangsters",
	["Mob boss"] = "Gangsters",
	["Civil Protection Chief"] = "Civil Protection",
	["Mayor"] = "Civil Protection"
},
entities = {
	["Drug lab"]    = "Other",
	["Money printer"]    = "Other",
	["Gun lab"]    = "Other"
},
shipments = {
	["AK47"] = "Rifles",
	["MP5"] = "Rifles",
	["M4"] = "Rifles",
	["Mac 10"] = "Other",
	["Pump shotgun"] = "Shotguns",
	["Sniper rifle"] = "Snipers"
},
weapons = {
	["Desert eagle"] = "Pistols",
	["Fiveseven"] = "Pistols",
	["Glock"] = "Pistols",
	["P228"] = "Pistols"
},
vehicles = {}, -- There are no default vehicles
ammo = {
	["Pistol ammo"]    = "Other",
	["Shotgun ammo"]    = "Other",
	["Rifle ammo"]    = "Other"
},
}
-- The list of weapons admins spawn with, in addition to the default weapons, a job's weapons and GM.Config.AdminCopWeapons
GM.Config.AdminWeapons = {
	""
}
-- Стандартные законы, из нельзя изменить во время игры.
GM.Config.DefaultLaws = {
	""
}
GM.Config.PocketBlacklist = {
	["fadmin_jail"] = true,
	["meteor"] = true,
	["door"] = true,
	["func_"] = true,
	["player"] = true,
	["beam"] = true,
	["worldspawn"] = true,
	["env_"] = true,
	["path_"] = true,
	["prop_physics"] = true,
	["money_printer"] = true,
	["gunlab"] = true,
	["prop_dynamic"] = true,
	["prop_vehicle_prisoner_pod"] = true,
	["keypad_wire"] = true,
	["gmod_button"] = true,
	["gmod_rtcameraprop"] = true,
	["gmod_cameraprop"] = true,
	["gmod_dynamite"] = true,
	["gmod_thruster"] = true,
	["gmod_light"] = true,
	["gmod_lamp"] = true,
	["gmod_emitter"] = true,
}
-- These weapons are classed as 'legal' in the weapon checker and are not stripped when confiscating weapons.
-- This setting is used IN ADDITION to GM.Config.weaponCheckerHideDefault and GM.Config.weaponCheckerHideNoLicense
-- You should use the former if you want to class the default weapons (GM.Config.DefaultWeapons and, if admin, GM.Config.AdminWeapons) and a player's job weapons as legal.
-- The latter takes GM.NoLicense weapons as legal (see licenseweapons.lua)
-- The format of this config is similar to GM.Config.DisallowDrop
GM.Config.noStripWeapons = {
}

-- The entities listed here will not be removed when a player changes their job.
-- This only applies when removeclassitems is set to true
-- Note: entities will only be removed when the player changes to a job that is not allowed to have the entity
GM.Config.preventClassItemRemoval = {
	["gunlab"] = false,
	["microwave"] = false,
	["spawned_shipment"] = false,
}
-- Properties set to true are allowed to be used. Values set to false or are missing from this list are blocked.
GM.Config.allowedProperties = {
	remover = true,
	ignite = false,
	extinguish = true,
	keepupright = true,
	gravity = true,
	collision = true,
	skin = true,
	bodygroups = true,
}
--[[---------------------------------------------------------------------------
F4 меню
---------------------------------------------------------------------------]]
-- hide the items that you can't buy and the jobs you can't get (instead of graying them out)
-- this option hides items when you don't have enough money, when the maximum is reached for a job or any other reason
GM.Config.hideNonBuyable = false
-- Hide only the items that you have the wrong job for (or for which the customCheck says no)
-- When you set this option to true and hideNonBuyable to false, you WILL see e.g. items that are too expensive for you to buy
-- but you won't see gundealer shipments when you have the citizen job
GM.Config.hideTeamUnbuyable = true
--[[---------------------------------------------------------------------------
AFK модуль
---------------------------------------------------------------------------]]
-- The time of inactivity before being demoted
GM.Config.afkdemotetime = 0
-- Prevent people from spamming AFK
GM.Config.AFKDelay = 300

--[[---------------------------------------------------------------------------
Модуль хитмена
---------------------------------------------------------------------------]]
-- The minimum price for a hit.
GM.Config.minHitPrice = 200
-- The maximum price for a hit.
GM.Config.maxHitPrice = 50000
-- The minimum distance between a hitman and his customer when they make the deal.
GM.Config.minHitDistance = 150
-- The text that tells the player he can press use on the hitman to request a hit.
GM.Config.hudText = "I am a hitman.\nPress E on me to request a hit!"
-- The text above a hitman when he's got a hit.
GM.Config.hitmanText = "Hit\naccepted!"
-- The cooldown time for a hit target (so they aren't spam killed).
GM.Config.hitTargetCooldown = 120
-- How long a customer has to wait to be able to buy another hit (from the moment the hit is accepted).
GM.Config.hitCustomerCooldown = 240

--[[---------------------------------------------------------------------------
Hungermod module
---------------------------------------------------------------------------]]
-- hungerspeed <Amount> - Set the rate at which players will become hungry (2 is the default).
GM.Config.hungerspeed = 0.4
-- starverate <Amount> - How much health that is taken away every second the player is starving  (3 is the default).
GM.Config.starverate = 1
